pkg/cleaner/cleaner.go

```diff
@@ -3,20 +3,19 @@ package cleaner
 
 import (
 	"log"
-	"os"
 
 	"k8s.io/kubernetes/pkg/api"
 
 	"github.com/deis/builder/pkg/k8s"
+	"github.com/deis/builder/pkg/sys"
 )
 
 const (
 	dotGitSuffix = ".git"
 )
 
-// Run starts the deleted app cleaner. Every pollSleepDuration, it compares the result of nsLister.List with the directories in the top level of gitHome on the local file system. On any error, it uses log messages to output a human readable description of what happened.
-func Run(gitHome string, nsLister k8s.NamespaceWatcher) error {
-
+// Run starts the deleted app cleaner. Every pollSleepD uration, it compares the result of nsLister.List with the directories in the top level of gitHome on the local file system. On any error, it uses log messages to output a human readable description of what happened.
```

Comment:
@smothiki these docs are out of date now. Can you fix them to reflect the fact that it listens to the event stream. Maybe something like this:

```
Run starts the deleted app cleaner. This function listens to the Kubernetes event stream for events that indicate namespaces that were `DELETED`. If the namespace name matches a folder on the local filesystem, this func deletes that folder. Note that this function blocks until the watcher returned by `nsLister.Watch` is closed, so you should launch it in a goroutine.
```